src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -688,7 +688,12 @@ object SbtBundle extends AutoPlugin {
     def toString[T](valueAndConfigName: (T, Option[String]), f: T => String): (String, Option[String]) =
       f(valueAndConfigName._1) -> valueAndConfigName._2
 
-    val componentPrefix = s"""components."${(normalizedName in config).value}""""
+    def escapeStartCommand(startCommand: Seq[String]): String = {
+      val executable = startCommand.head
+      val arguments = startCommand.tail
+      val executableEscaped = executable.replace("\\", "/")
+      formatSeq(Seq(executableEscaped) ++ arguments)
+    }
```

Comment:
@huntc - if we were to do this, that means when `BundleKeys.startCommand` is overridden or appended in the `build.sbt`, the new settings won't have the correct value.

I'm not sure how to link the new settings to the `BundleKeys.startCommand` so when it the `BundleKeys.startCommand` is updated, the new settings has the correct value as well.